radio/src/lua/api_general.cpp

```diff
@@ -1129,6 +1135,83 @@ static int luaAccessTelemetryPush(lua_State * L)
 #endif
 
 #if defined(CROSSFIRE)
+#if defined(COLORLCD)
+static int luaCrossfireTelemetryRemovePrivateQueue(lua_State * const L) 
+{
+  const int id = luaL_checkinteger(L, 1);
+  TRACE("crossfireTelemetryRemovePrivateQueue: %d", id);
+  luaTelemetryQueueMgr.remove(id);
+  return 0;
+}
+
+static int luaCrossfireTelemetryCreatePrivateQueue(lua_State * const L)
```

Comment:
This should actually return a LUA object. `luaCrossfireTelemetryRemovePrivateQueue` should probably be declared as `__gc` for that object table, which would be called when the object is garbage collected.